README.md

```diff
@@ -1,2 +1,45 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+int MAX_GUESSES = 4
+string TO_GUESS = "Word to guess"
+bool won = False
+print("Guess the " + TO_GUESS.length() + " letter word")
+print("_" * TO_GUESS)
+for(int y = 0; y < MAX_GUESSES; y++)
+{
+  int counter = 0
+  bool chk = True
```

Comment:
`chk` is not a very good name -- short for "check" presumably, but checking what?